downstream/assemblies/security/assembly-idps.adoc

```diff
@@ -16,25 +16,25 @@ ifdef::context[:parent-context: {context}]
 The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken. Include {context} in the ID so the assembly can be reused.
 ////
 
-[id="assembly-intrusion-detection-system_{context}"]
+[id="assembly-idps_{context}"]
 
-= Automating Network Intrusion Detection System (IDS) Rules with Ansible
+= Automating Network Intrusion Detection and Prevention Systems (IDPS) with Ansible
 
-:context: instrusion-detection-system
+:context: idps
 
 [role="_abstract"]
 
-You can use Ansible to automate your Intrusion Detection System (IDS). For the purpose of this guide, we use Snort as the IDS. Use Ansible automation hub to consume content collections, such as tasks, roles, and modules to create automated workflows.
+You can use Ansible to automate your Intrusion Detection and Prevention Systems (IDPS). For the purpose of this guide, we use Snort as the IDS. Use Ansible automation hub to consume content collections, such as tasks, roles, and modules to create automated workflows.
```

Comment:
`You can use Ansible to automate your Intrusion Detection and Prevention Systems (IDPS). For the purpose of this guide, we use Snort as the IDS.`

to 

`You can use Ansible to automate your Intrusion Detection and Prevention System (IDPS). For the purpose of this guide, we use Snort as the IDPS.`